### ISSUE TITLE
Different textures for healthpacks on different machines

### ISSUE THREAD

**Issue by edbeeching** [mwydmuch/ViZDoom#284]
I have VizDoom 1.1.5 install on two machines both running Ubuntu 16.04.
One has a GPU and does not, although I am not sure if this is related to the problem.
On one machine I observed that the texture used for the Healthpack in the health gathering scenario is different to that of the other machine.
Here are examples from the two machines, sorry for the poor resolution I run at 160x120 and downsample by 2 for training a RL agent.
[Machine without GPU](https://imgur.com/a/9m2Go)
[Machine with GPU](https://imgur.com/a/QEx38)

As I am training a RL agent I would like to be able to train on the GPU machine and occasional test the model on the other machine. Any help with this matter would be much appreciated.

**Comment by mwydmuch** [comment]
Hi @edbeeching,
your machine without gpu has older Freedoom version (v0.10), the one with gpu has the latest version (v0.11) (we updated Freedoom version in ViZDoom in October). Just reinstall ViZDoom on your machine without gpu.

**Comment by edbeeching** [comment]
@mwydmuch 
Thank you for the fast response.
It is funny because I only installed this on both machines over the last 2 weeks.
I am using the newest version from the repo in both cases and I just reinstalled on both and still observe the same issue.
You can find the outputs of the install processes here (using pip verbose mode)
[Paste from machine without GPU](https://pastebin.com/ePS5CLUW)
[Paste from machine with GPU](https://pastebin.com/aunsnUgk)

Any help would be much appreciated.

**Comment by mwydmuch** [comment]
Sorry @edbeeching for lack of further response yesterday.
ViZDoom tries to load main flaws file from working directory first, maybe you have here some old version of freedoom2.wad? This is the only explanation I can think of now.

**Comment by edbeeching** [comment]
@mwydmuch 
Thanks again for your reply. I looked into it further and for some reason my IDE (Spyder) was not loading vizdoom from my specified virtualenv, but from the standard python environment which contained version 1.1.4.
I removed the vizdoom from the base environment and the problem is solved. So all in all I was just importing the old version without realizing.

Thank you for your help and sorry to bother you. Feel free to close the issue.
